programs/mango-v4/src/health/cache.rs

```diff
@@ -258,7 +294,13 @@ impl Serum3Info {
         let quote_asset = quote_info.prices.asset(health_type);
         let base_liab = base_info.prices.liab(health_type);
         // OPTIMIZATION: These divisions can be extremely expensive (up to 5k CU each)
-        self.reserved_base + self.reserved_quote * quote_asset / base_liab
+        let reserved_quote_as_base_oracle = self.reserved_quote * quote_asset / base_liab;
```

Comment:
what would be the impact of calculating this in f64, given that reserved_quote_as_base_highest_bid is stored in f64 anyways?

Reply:
I suspect that f64 computation would be fine. The error in the oracle price should way bigger than any rounding. Measuring and testing this had been a possibility (hence the "// optimization" comments), but serum3 health contributions haven't been a limiting compute factor so far.